README.md

```diff
@@ -86,11 +86,13 @@ need them.
 While Rustls itself is platform independent it uses
 [`ring`](https://crates.io/crates/ring) for implementing the cryptography in
 TLS. As a result, rustls only runs on platforms
-supported by `ring`. At the time of writing this means x86, x86-64, armv7, and
-aarch64. For more information see [the supported `ring` CI
-targets](https://github.com/briansmith/ring/blob/9cc0d45f4d8521f467bb3a621e74b1535e118188/.github/workflows/ci.yml#L151-L167).
+supported by `ring`. At the time of writing this means x86, x86-64, aarch64,
+armv7, powerpc64le, riscv64gc and s390x. For more information, see [the
```

Comment:
The targets supported are documented in code in https://github.com/briansmith/ring/blob/main/include/ring-core/target.h#L25

32-bit ARM, Aarch64 (64-bit ARM), X86, x86-64, LOONGARCH64, 32-bit & 64-bit little-endian MIPS, 32-bit PowerPC (Big Endian), 64-bit PowerPC (Big and Little Endian),  64-bit RISC-V, s390x, and WebAssembly.

Reply:
Thanks :+1: This should be fixed in `main`: https://github.com/rustls/rustls/pull/1532 I'll backport once that's done.

Reply:
Update in both `main` and this branch.